specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5628 @@
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "skipCount",
+            "in": "query",
+            "description": "The skip count, which specifies the number of elements that can be bypassed from a sequence and then return the remaining elements.",
+            "required": false,
+            "type": "integer"
+          },
+          {
+            "name": "takeCount",
+            "in": "query",
+            "description": "The take count , which specifies the number of elements that can be returned from a sequence.",
+            "required": false,
+            "type": "integer"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Services"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/operations": {
+      "get": {
+        "tags": [
+          "operations"
+        ],
+        "x-ms-examples": {
+          "operations_list": {
+            "$ref": "./examples/OperationList.json"
+          }
+        },
+        "operationId": "operations_list",
+        "description": "Lists the available Azure Data Factory API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/configuration": {
+      "post": {
+        "tags": [ "Configuration" ],
+        "description": "Onboards a tenant in Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "configuration_add": {
+            "$ref": "./examples/Tenant.json"
+          }
+        },
+        "operationId": "configuration_add",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Tenant"
+            }
+          },
+          "403": {
+            "description": "The tenant is not authorized to onboard to Azure Active Directory Connect Health because it does not meet the onboarding criteria."
+          }
+        }
+      },
+      "get": {
+        "tags": [ "Configuration" ],
+        "description": "Gets the details of a tenant onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "configuration_get": {

```

Comment:
Wondering if this should be "tenant_get"? Not saying it necessarily *should* be - you better know the terminology associated with your service - just wondering :)